### ISSUE TITLE
workbench.action.terminal.* shortcuts override other shortcuts

### ISSUE THREAD

**Issue by MBAatEJOT** [microsoft/vscode#123085]
Issue Type: <b>Bug</b>

At least some shortcuts for the terminal overrides all the other shortcuts.

This problem appears at least for the schort cuts "delete" and "f2".

## "delete":
I was using a multi cursor and have selected a specific word. When I hit the "delete" button on my keyboard it focused the terminal instead of deleting the selected word. Even the terminal didn't had the focus.
After I removed the binding from workbench.action.terminal.killInstance the default functionality was restored.

## "f2":
I try to rename a method in C# and therefor hitting "f2" but nothing happens. Only after I removed the key binding for workbench.action.terminal.renameInstance the default functionality was restored.

Those are the two shortcuts I have already used. At the moment I discovered no other problems with not working shortcuts. But I will add them as soon as they appears.

## System information
VS Code version: Code 1.56.0 (cfa2e218100323074ac1948c885448fdf4de2a7f, 2021-05-04T22:09:06.405Z)
OS version: Windows_NT x64 10.0.18363
Remote OS version: Linux x64 4.19.128-microsoft-standard

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6440HQ CPU @ 2.60GHz (4 x 2592)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.88GB (4.74GB free)|
|Process Argv|--folder-uri vscode-remote://dev-container%2B7b227265706f7369746f727950617468223a2268747470733a2f2f454a4f542d446576406465762e617a7572652e636f6d2f454a4f542d4465762f454a4f63616c632f5f6769742f454a4f63616c63222c22766f6c756d654e616d65223a225f5f554e495155455f5f222c22666f6c646572223a22454a4f63616c63222c22636c6f6e65496e666f223a7b2275726c546f436c6f6e65223a2268747470733a2f2f454a4f542d446576406465762e617a7572652e636f6d2f454a4f542d4465762f454a4f63616c632f5f6769742f454a4f63616c63227d2c22696e7370656374566f6c756d65223a66616c73657d/workspaces/EJOcalc --crash-reporter-id 5ce13f2e-761a-433b-b292-fd8a986c0c83|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|Dev Container: EJOcalc|
|OS|Linux x64 4.19.128-microsoft-standard|
|CPUs|Intel(R) Core(TM) i5-6440HQ CPU @ 2.60GHz (4 x 2592)|
|Memory (System)|12.39GB (5.96GB free)|
|VM|0%|
</details><details><summary>Extensions (19)</summary>

Extension|Author (truncated)|Version
---|---|---
codespaces|Git|0.10.0
vscode-peacock|joh|3.9.1
remote-containers|ms-|0.177.0
remote-ssh|ms-|0.65.4
remote-ssh-edit|ms-|0.65.4
remote-wsl|ms-|0.56.1
vscode-remote-extensionpack|ms-|0.20.0
debugger-for-chrome|msj|4.12.12
vscode-eslint|dba|2.1.20
xml|Dot|2.5.1
gitlens|eam|11.4.1
todo-tree|Gru|0.0.213
beautify|Hoo|1.5.0
docomment|k--|0.1.20
vscode-docker|ms-|1.12.1
csharp|ms-|1.23.11
sqltools|mtx|0.23.0
sqltools-driver-mssql|mtx|0.2.0
quicktype|qui|12.0.46

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
pythonvspyt602cf:30294773
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvsnew554cf:30291488
pythontb:30283811
vspre833cf:30267465
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158cf:30286554
vscgsv2ct:30294353

```

</details>

**Comment by gjsjohnmurray** [comment]
See also #123040

**Comment by meganrogge** [comment]
Duplicate of #123040
